int32 ind1
int32 ind2
---
string txt1
string txt2
float64 val1
float64 val2

